ouroboros-consensus/src/Ouroboros/Consensus/HardFork/Combinator/Ledger.hs

```diff
@@ -215,6 +215,13 @@ instance (SingleEraBlock x, TableStuff (LedgerState x)) => TableStuff (LedgerSta
   mapLedgerTables  f = coerce $ mapLedgerTables  @(LedgerState x) f
   zipLedgerTables  f = coerce $ zipLedgerTables  @(LedgerState x) f
   foldLedgerTables f = coerce $ foldLedgerTables @(LedgerState x) f
+  traverseLedgerTables f (LedgerTablesOne tbls)
+    = LedgerTablesOne <$> traverseLedgerTables f tbls
+  zip2ALedgerTables f
```

Comment:
Minor: I would expect `zipLedgerTablesA` based on the existing patterns in `base` eg